modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/MediaDescriptors/Extensions/MediaDescriptorExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.IO;
+using System.Linq;
+
+namespace Volo.CmsKit.MediaDescriptors.Extensions
+{
+    public static class MediaDescriptorExtensions
+    {
+        public static bool IsValidMediaFileName(this string name)
```

Comment:
We should find a better way to do this. Can you do research and present it to the team before implementation?